docs/docker/README.md

```diff
@@ -0,0 +1,184 @@
+# Deploy with Docker
+
+## Prerequisites
+
+Docker provides a way to run applications securely isolated in a container, packaged with all its dependencies and libraries.
+To learn more on Docker, kindly refer here: [What is Docker?](https://www.docker.com/what-docker).
+
+Make sure that you have the latest version of Docker installed on your machine. [Get latest version](https://www.docker.com/products/overview#/install_the_platform)
+
+[Notes for non-Linux systems](notes-for-non-linux.md)
+
+## Usage instructions ##
+
+### Step 1 - Setup network ###
+Create a network in order for the cBioPortal container and mysql database to communicate.
+```
+docker network create cbio-net
+```
+
+### Step 2 - Run mysql with seed database ###
+Start a MySQL server. The command below stores the database in a folder named
+`/<path_to_save_mysql_db>/db_files/`. This should be an absolute path.
+
+```
+docker run -d --restart=always \
+  --name=cbioDB \
+  --net=cbio-net \
+  -e MYSQL_ROOT_PASSWORD='P@ssword1' \
+  -e MYSQL_USER=cbio \
+  -e MYSQL_PASSWORD='P@ssword1' \
+  -e MYSQL_DATABASE=cbioportal \
+  -v /<path_to_save_mysql_db>/db_files/:/var/lib/mysql/ \
+  mysql:5.7
+```
+
+Download the seed database from the
+[cBioPortal Datahub](https://github.com/cBioPortal/datahub/blob/master/seedDB/README.md),
+and use the command below to upload the seed data to the server started above.
+
+Make sure to replace
+`/<path_to_seed_database>/seed-cbioportal_<genome_build>_<seed_version>.sql.gz`
+with the path and name of the downloaded seed database. Again, this should be
+an absolute path.
+
+```
+docker run \
+  --name=load-seeddb \
+  --net=cbio-net \
+  -e MYSQL_USER=cbio \
+  -e MYSQL_PASSWORD='P@ssword1' \
+  -v /<path_to_seed_database>/cgds.sql:/mnt/cgds.sql:ro \
+  -v /<path_to_seed_database>/seed-cbioportal_<genome_build>_<seed_version>.sql.gz:/mnt/seed.sql.gz:ro \
+  mysql:5.7 \
+  sh -c 'cat /mnt/cgds.sql | mysql -hcbioDB -u"$MYSQL_USER" -p"$MYSQL_PASSWORD" cbioportal \
+      && zcat /mnt/seed.sql.gz |  mysql -hcbioDB -u"$MYSQL_USER" -p"$MYSQL_PASSWORD" cbioportal'
+```
+
+Follow the logs of this step to ensure that no errors occur. If any error
+occurs, make sure to check it. A common cause is pointing the `-v` parameters
+above to folders or files that do not exist.
+
+### Step 3 - Set up a portal.properties file ###
+
+Copy the
+[`portal.properties.EXAMPLE`](../../src/main/resources/portal.properties.EXAMPLE)
+and change it according to your wishes. See the [full
+reference](portal.properties-Reference.md) and the [skin
+properties](Customizing-your-instance-of-cBioPortal.md) for more information on
+the relevant properties.
+
+Make sure to at least provide the database parameters from step 1, which are
+required for the next step:
+
+```
+db.user=cbio
+db.password=P@ssword1
+db.host=cbioDB
+db.portal_db_name=cbioportal
+db.connection_string=jdbc:mysql://cbioDB/
+```
```

Comment:
please document the use of db.use_ssl option, it will need to be set to **_true_** if your mysql server use SSL. 

> db.use_ssl = true 

or use JAVA_OPTS when run docker 

> -Ddb.use_ssl=true

Reply:
included that

Reply:
thank you, @inodb!